File: Modules/HumanResource/EmployeeLate/EmployeeLate/EmployeeLateForm.cs
using Serenity.ComponentModel;
using System;

namespace HRMSoftware.EmployeeLate.Forms;

[FormScript("EmployeeLate.EmployeeLate")]
[BasedOnRow(typeof(EmployeeLateRow), CheckNames = true)]
public class EmployeeLateForm
{
    public int EmployeeRowId { get; set; }
    public string EmployeeName { get; set; }
    public DateTime Date { get; set; }
    public int LateMins { get; set; }
}

File: Modules/ServerTypes/EmployeeLate/EmployeeLateForm.ts
import { LookupEditor, StringEditor, DateEditor, IntegerEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface EmployeeLateForm {
    EmployeeRowId: LookupEditor;
    EmployeeName: StringEditor;
    Date: DateEditor;
    LateMins: IntegerEditor;
}

export class EmployeeLateForm extends PrefixedContext {
    static formKey = 'EmployeeLate.EmployeeLate';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!EmployeeLateForm.init)  {
            EmployeeLateForm.init = true;

            var w0 = LookupEditor;
            var w1 = StringEditor;
            var w2 = DateEditor;
            var w3 = IntegerEditor;

            initFormType(EmployeeLateForm, [
                'EmployeeRowId', w0,
                'EmployeeName', w1,
                'Date', w2,
                'LateMins', w3
            ]);
        }
    }
}